system-tests/services/test-pods-communication.js

```diff
@@ -0,0 +1,77 @@
+require("../_support/utils/ServicesUtil");
```

Comment:
Are we starting to do one test per file? `test-pods-communication.js` this is way too specific I believe. In this file we are testing we can edit the pod using the json format.